deploy/environments/externals.yaml

```diff
@@ -23,6 +23,7 @@ spack:
   specs:
     - llvm@11.0.0 +python
     - nvhpc@20.9 install_type=network
+    - nvhpc@21.2 install_type=network
```

Comment:
Just a note : we are going to keep 20.9 until we solve some issues with latest versions.

Reply:
k